docs/filemanagement.rst

```diff
@@ -260,3 +328,4 @@ it you indicate that you acknowledge the risks of downloading potential malware.
 .. _`official documentation`: https://developers.google.com/drive/v2/reference/files#resource-representations
 .. _`known`: https://productforums.google.com/forum/#!topic/docs/BJLimQDGtjQ
 .. _`abusive`: https://support.google.com/docs/answer/148505
+.. _`q_parameters`: https://developers.google.com/drive/api/guides/search-files#examples
```

Comment:
why `q_paramaters` (what does this name mean?)

Reply:
short for query parameters, i will rename it to query_parameters as it's more descriptive

Reply:
make sense, so I think in the doc they call `q` itself a parameter and it can take different values. Can we just explicitly say in the text `Query string examples` instead?

Reply:
done